<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDoubleCameraDlg</class>
 <widget class="QDialog" name="CDoubleCameraDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>双目相机</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_LeftImage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 127, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_RightImage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 127, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>127</width>
           <height>114</height>
          </rect>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_capImg">
            <property name="text">
             <string>拍摄原图</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBox_GenDepthImg">
            <property name="text">
             <string>深度图</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_CorrectImg">
            <property name="text">
             <string>校正图</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkBox_GenPlyImg">
            <property name="text">
             <string>点云图</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_LeftCorrectImage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 127, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_RightCorrectImage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 127, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_DepthImage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 127, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_PlyImage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 127, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
